src/components/header/header.tsx

```diff
@@ -47,14 +59,34 @@ export const Header: FC<IHeaderProps> = ({ role = "all", onLogin }) => {
             </li>
           ))}
       </NavigationWrapper>
-      <ButtonWrapper>
-        <Button variant="outlined" size="small" onClick={onLogin}>
-          Log in
-        </Button>
-        <Button size="small" color="secondary">
-          Sign up
-        </Button>
-      </ButtonWrapper>
+      {data?.user?.name ? (
+        <ButtonWrapper>
+          <p>Welcome back, {data?.user?.name}</p>
+          <Button
+            variant="outlined"
+            size="small"
+            color="secondary"
+            onClick={() => signOut()}
+          >
+            Sign out
```

Comment:
🦆 we should declare a constants file somwhere in the project where we can store this strings, and then reference them in the files :)

Reply:
🥔 I created a constants file and updated all texts to use constants instead 📐